client/src/routes/temperaturesensor/temperature.js

```diff
@@ -0,0 +1,313 @@
+import React, { Component } from 'react';
+import withStyles from 'isomorphic-style-loader/withStyles';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { connect } from 'react-redux';
+import { Spinner, Alert } from 'react-bootstrap';
+import { Line, defaults } from 'react-chartjs-2';
+import getChartData from './chart-dataset';
+import Icon from './temperatureIcon/temperature-icon';
+import Loader from '../../components/Loader/Loader';
+import { setFilter, changeTemperatureStatus } from '../../actions/temperature';
+import style from './Temperature.css';
+import TemperatureContainer from './TemperatureContainer/TemperatureContainer';
+
+defaults.global.defaultFontFamily = 'Montserrat';
+
+const buttonsStyle = {
+  marginLeft: '5.5em',
+  marginTop: '1em',
+};
+
+const xAxeStyle = {
+  font: '20px serif',
+  paddingLeft: '17em',
+};
+
+const defaultButt = 'btn btn-secondary btn-lg';
```

Comment:
Rename this. https://context.reverso.net/%D0%BF%D0%B5%D1%80%D0%B5%D0%B2%D0%BE%D0%B4/%D0%B0%D0%BD%D0%B3%D0%BB%D0%B8%D0%B9%D1%81%D0%BA%D0%B8%D0%B9-%D1%80%D1%83%D1%81%D1%81%D0%BA%D0%B8%D0%B9/butt